### DISCUSSION TITLE
How to setup weblate 5.x for markdown ?

### DISCUSSION THREAD

**Discussion by rungeard** [WeblateOrg/weblate#10467] [Q&A]
Hi everyone !

I saw that weblate 5 is supporting support for markdown files. 
I have a git repo with the the following structure : 
```
home.md
page1.md
└── fr
    ├── home.md
    └── page1.md
└── de
    ├── home.md
    └── page1.md
```
 I try without success to setup weblate to work with this structure. Is it possible ? 

Thanks in advance for your feedbacks !

**Comment by nijel** [comment]
Create first component for `home.md` manually and then configure [Component dicscovery add-on](https://docs.weblate.org/en/latest/admin/addons.html#component-discovery) to add other markdown files.
